pkg/domain/entities/containers.go

```diff
@@ -44,6 +49,16 @@ type StopReport struct {
 	Id  string
 }
 
+type TopOptions struct {
+	// CLI flags.
+	ListDescriptors bool
+	Latest          bool
```

Comment:
If these are only for the CLI, then they should remain in top.go.

Reply:
Not sure this would make a difference. The API is shared and the options are already a parameter. If we remove it here, we had to add it as another parameter.